packages/opam-devel/opam-devel.2.0.8/opam

```diff
@@ -17,15 +17,19 @@ authors: [
 ]
 homepage: "https://opam.ocaml.org"
 bug-reports: "https://github.com/ocaml/opam/issues"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
 depends: [
   "ocaml" {>= "4.02.3"}
   "opam-client" {= version}
-  "cmdliner" {>= "0.9.8"}
+  "cmdliner" {>= "1.0.0"}
   "dune" {>= "1.2.1"}
+  "conf-openssl" {with-test}
+  "conf-diffutils" {with-test}
 ]
 build: [
   ["./configure" "--disable-checks" "--prefix" prefix]
   [make "%{name}%.install"]
+  [make "tests"] {with-test}
```

Comment:
This was removed on purpose. See https://github.com/ocaml/opam/pull/4500

Reply:
I thought it was removed from 2.1 on purpose - does it affect 2.0 as well (different testing system)?

Reply:
yes, it is the old testing system in 2.0

Reply:
It still requires a system compiler, even if no network. There's no totally principled way in opam-repository at the moment of depending on the presence of a _system_ compiler itself (it's not that we need to depend on `ocaml-system`, it's that we actually need the depext packages for ocaml installed) so given that tests in opam-devel 2.1 later can't be run in opam anyway, I've removed this.